src/herbpy/tsr/box.py

```diff
@@ -0,0 +1,47 @@
+import numpy
+import prpy.tsr
+
+@prpy.tsr.tsrlibrary.TSRFactory('herb', 'box', 'stamp')
+def box_stamp(robot, box, manip=None):
+
+    '''
+    This creates a TSR for stamping a box. 
+    It is assumed that when called, the robot is grasping a stamp
+
+    @param robot The robot to perform the stamp
+    @param box The box to stamp
+    @param manip The manipulator to stamp 
+    '''
+    
+    with robot.GetEnv():
```

Comment:
This lock is not necessary. The environment is guaranteed to be locked around a `tsrlibrary` call.

@jeking04, can you confirm?